### PR TITLE
CRA-69 - Examination Module Completed

### PR DESCRIPTION
**Pull request by shresthdeepgupta**

### CODE REVIEW

File: android/MaterialDesignTest/app/src/test/java/com/testexample/materialdesigntest/data/network/repository/UserRemoteRepositoryTest.kt [hunk 1]
```diff
@@ -86,38 +101,36 @@ class UserRemoteRepositoryTest {
...
         assertEquals(validStudentLoginRequest.code, student.studentCollegeCode)
     }
 
 
     @Test
     fun `when unregistered student roll number is supplied for login`() {
         val output = repository
-                .authStudent(StudentLoginRequest("168021067",
-                        validStudentLoginRequest.code, validStudentLoginRequest.password))
+            .authStudent(StudentLoginRequest("168021067",
```
Comment (akanksha-einn) [inline android/MaterialDesignTest/app/src/test/java/com/testexample/materialdesigntest/data/network/repository/UserRemoteRepositoryTest.kt:125]: Remove this hard coded roll number

File: android/MaterialDesignTest/app/src/test/java/com/testexample/materialdesigntest/data/network/repository/ExaminationRemoteRepoTest.kt [hunk 2]
```diff
@@ -146,8 +144,9 @@ class ExaminationRemoteRepoTest {
 
     @Test
     fun callApiForEndingExam() {
+        println(studentSession.id + studentSession.token)
         val output = repository
-            .callApiForEndingExam(EndExamRequest(studentSession.id,
+            .callApiForEndingExam(studentSession.token,EndExamRequest(studentSession.id,
```
Comment (akanksha-einn) [inline android/MaterialDesignTest/app/src/test/java/com/testexample/materialdesigntest/data/network/repository/ExaminationRemoteRepoTest.kt:149]: add space between parameters

File: android/MaterialDesignTest/app/src/main/res/layout/fragment_exam.xml [hunk 3]
```diff
@@ -18,40 +18,20 @@
...
-        <com.google.android.material.tabs.TabItem
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:text="3" />
-    </com.google.android.material.tabs.TabLayout>
+<!--    <androidx.viewpager.widget.ViewPager-->
+<!--        android:layout_width="match_parent"-->
+<!--        android:layout_height="match_parent"-->
+<!--        android:id="@+id/tabLayoutFrame"-->
+<!--        android:background="@color/colorAccent2"/>-->
```
Comment (akanksha-einn) [inline android/MaterialDesignTest/app/src/main/res/layout/fragment_exam.xml:34]: Remove comments

File: android/MaterialDesignTest/app/src/main/java/com/testexample/materialdesigntest/ui/instructions/InstructionsFragment.kt [hunk 4]
```diff
@@ -1,52 +1,58 @@
...
+import com.testexample.materialdesigntest.data.model.Student
 import com.testexample.materialdesigntest.ui.ProgressBar
 import com.testexample.materialdesigntest.ui.examination.ExamDrawer
+import com.testexample.materialdesigntest.utils.Constants
 import kotlinx.android.synthetic.main.fragment_instructions.*
+import com.testexample.materialdesigntest.ui.instructions.TAG as InstructionsTAG
 
-private const val QUESTION_PAPER_ID = "question_paper_id"
-private const val INSTRUCTION_ID = "instructions_id"
+private const val TAG = "Instruction Fragment"
```
Comment (akanksha-einn) [inline android/MaterialDesignTest/app/src/main/java/com/testexample/materialdesigntest/ui/instructions/InstructionsFragment.kt:19]: we have imported "TAG" as "InstructionsTAG" but haven't use the same

File: android/MaterialDesignTest/app/src/main/java/com/testexample/materialdesigntest/ui/instructions/InstructionsFragment.kt [hunk 5]
```diff
@@ -1,52 +1,58 @@
...
+            questionPaper = it.getParcelable(Constants.QUESTION_PAPER)!!
+            student = it.getParcelable(Constants.STUDENT)!!
         }
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        Log.d(TAG," on viewCreated")
+        Log.d(InstructionsTAG," on viewCreated")
```
Comment (shresthdeepgupta) [inline android/MaterialDesignTest/app/src/main/java/com/testexample/materialdesigntest/ui/instructions/InstructionsFragment.kt:40]: we have used it here

File: android/MaterialDesignTest/app/src/main/java/com/testexample/materialdesigntest/ui/instructions/InstructionsFragment.kt (hunk 4 above)
Comment (shresthdeepgupta) [reply android/MaterialDesignTest/app/src/main/java/com/testexample/materialdesigntest/ui/instructions/InstructionsFragment.kt:19]: we have used

**Pull request merged**